api/src/main/java/org/pmiops/workbench/utils/WorkspaceMapper.java

```diff
@@ -104,66 +108,69 @@ void mergeResearchPurposeIntoWorkspace(
   WorkspaceResource workspaceResourceFromDbWorkspace(
       DbWorkspace dbWorkspace, WorkspaceAccessLevel accessLevel);
 
+  @Mapping(target = "workspaceId", source = "dbWorkspace.workspaceId")
   @Mapping(target = "workspaceFirecloudName", source = "dbWorkspace.firecloudName")
   @Mapping(target = "workspaceBillingStatus", source = "dbWorkspace.billingStatus")
   @Mapping(target = "permission", source = "accessLevel")
-  @Mapping(target = "cohort", source = "cohort")
+  @Mapping(target = "cohort", source = "dbCohort")
   // All workspaceResources have one object and all others are null. That should be
   // defined by a setter where used
   @Mapping(target = "cohortReview", ignore = true)
   @Mapping(target = "conceptSet", ignore = true)
   @Mapping(target = "dataSet", ignore = true)
   @Mapping(target = "notebook", ignore = true)
   // This should be set when the resource is set
-  @Mapping(target = "modifiedTime", source = "cohort.lastModifiedTime")
-  WorkspaceResource workspaceResourceFromDbWorkspaceAndCohort(
-      DbWorkspace dbWorkspace, WorkspaceAccessLevel accessLevel, Cohort cohort);
+  @Mapping(target = "modifiedTime", source = "dbCohort.lastModifiedTime")
+  WorkspaceResource workspaceResourceFromDbWorkspaceAndDbCohort(
```

Comment:
Our emerging naming convention for these appears to be xAndYToZ(), not zFromXAndY().